tests/time_tests/include/timetests_helper/timer.h

```diff
@@ -17,16 +17,17 @@ When destroyed, reports duration.
 class Timer {
 private:
   std::string name;
+  int order;
   time_point start_time;
 
 public:
   /// Constructs Timer object and measures start time.
-  Timer(const std::string &timer_name);
+  Timer(const std::string &timer_name, int order_number);
```

Comment:
order number is defined by calling of `Timer` constructor, so there may be a global counter which will be increased each time `Timer` is created